### DISCUSSION TITLE
How can I see which branches are not covered?

### DISCUSSION THREAD

**Discussion by SimonSimCity** [codecov/feedback#78] [General]
One of the files in a project I'm working with has a line-coverage of 100%, but a branch-coverage of 97%.

The report just shows all lines as green. How can I see which branches are not covered yet?

Here's one of the files this happens in a project I'm working on: https://app.codecov.io/gh/bcc-code/bmm-web/blob/main/plugins%2FmediaPlayer%2FQueue.ts

**Comment by rohan-at-sentry** [comment]
Hey @SimonSimCity - not sure what's going on here, I'm opening an issue so we can get you some support. I noticed one other instance of this [here](https://app.codecov.io/gh/bcc-code/bmm-web/commit/2f0808ba4f0d1ff7d232dbc239c9a365c0a1a04b/blob/plugins/mediaPlayer/mediaPlayer.ts)

**Reply by rohan-at-sentry** [reply]
Issue https://github.com/codecov/feedback/issues/79

**Reply by SimonSimCity** [reply]
@rohan-at-sentry has this been closed because the conversation got moved? At least I still have this issue (https://app.codecov.io/gh/bcc-code/bmm-web/blob/main/plugins%2FmediaPlayer%2FQueue.ts)...

**Reply by rohan-at-sentry** [reply]
Hey @SimonSimCity - that's right, this issue https://github.com/codecov/feedback/issues/79 is how we can track.
